Services/CryptionService.cs

```diff
@@ -0,0 +1,36 @@
+﻿using LethalCompany_Backend.Services.Interfaces;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace LethalCompany_Backend.Services;
+
+public class CryptionService : ICryptionService
+{
+    public string Decrypt(string key)
+    {
+        try
```

Comment:
Why Try catch? you returning the exception mesasge as decrypted value? You know this can go wrong?